UnityProject/Assets/Scripts/Managers/SubSceneManager/SubSceneManager.Client.cs

```diff
@@ -49,4 +51,36 @@ IEnumerator LoadClientSubScene(SceneInfo sceneInfo)
 
 		clientIsLoadingSubscene = false;
 	}
+
+	public void LoadScenesFromServer(List<SceneInfo> Scenes, string OriginalScene, Action OnFinish)
+	{
+		StartCoroutine(LoadClientScenesFromServer(Scenes,OriginalScene, OnFinish));
+	}
+
+	IEnumerator LoadClientScenesFromServer(List<SceneInfo> Scenes, string OriginalScene, Action OnFinish)
+	{
+		clientIsLoadingSubscene = true;
+		foreach (var Scene in Scenes)
+		{
+			clientIsLoadingSubscene = true;
+			yield return LoadClientSubScene(Scene, false);
+			clientIsLoadingSubscene = true;
```

Comment:
Why twice?

Reply:
just to make double sure

Reply:
Well actually thrice, to make triple sure? Why would it change before we want it to?

Reply:
because the scene manager will try loading in other scenes, while this is loading in scenes